index.html

```diff
@@ -180,14 +211,125 @@ <h2>Debug Zone</h2>
         <!-- Here we try to reach the streamlabs API to see if we can get your
         donation goal progress manually. "module" b/c we use `await` -->
         <script type="module">
-            const url =
-                "https://streamlabscharity.com/api/v1/users/campaigns/schmidlak/2024-birthday-stream-for-alzheimers-awareness-and-research";
-            const response = await fetch(url);
-            console.log("Fetch response");
-            console.log(response);
-            const body = await response.json();
-            document.getElementById("fetch-results").textContent =
-                JSON.stringify(body, null, 4);
+            try {
+                const url =
+                    "https://streamlabscharity.com/api/v1/users/campaigns/schmidlak/2024-birthday-stream-for-alzheimers-awareness-and-research";
+                const response = await fetch(url);
+                console.log("Fetch response:", response);
+                const data = await response.json();
+                // debug view in debug zone
+                // document.getElementById("fetch-results").textContent =
+                //     JSON.stringify(data, null, 4);
+                const raisedCents = data["amount_raised"];
+                // const raisedCents = 10000; // uncomment to try experimental value
+                const goalCents = data["goal"]["amount"]; // should be 90000 for $900
+                const donoProgressEl = document.getElementById("dono-progress");
+                donoProgressEl.setAttribute(
+                    "value",
+                    Math.min(100, (100 * raisedCents) / goalCents).toFixed(1)
+                );
+                const donoStatusEl = document.getElementById("dono-status");
+                donoStatusEl.textContent = "$" + (raisedCents / 100).toFixed(2);
+            } catch (error) {
+                console.error(
+                    "Error fetching and setting donation progress:",
+                    error
+                );
+            }
+        </script>
+        <script type="module">
+            // 1. Fixed time Nov 10 2024 11am PST
+            const startTime = new Date("2024-11-10T11:00:00-08:00");
+            // 2. Fixed time Nov 10 2024 11pm PST (add 12 hours)
+            const endTime = new Date("2024-11-10T23:00:00-08:00");
+
+            function updateTime() {
+                // 3. Current time
+                const now = new Date();
+                const beforeStream = new Date("2024-11-10T09:00:00-08:00");
+                const duringStream = new Date("2024-11-10T13:00:00-08:00");
+                const afterStream = new Date("2024-11-10T23:30:00-08:00");
+                const currentTime = now; // pick different ones here to test
```

Comment:
change `now` to any of the other variables here to see how stuff looks at different times